src/java/org/apache/cassandra/net/InboundMessageHandler.java

```diff
@@ -822,7 +828,7 @@ private Message deserialize()
             catch (IncompatibleSchemaException e)
             {
                 callbacks.onFailedDeserialize(size, header, e);
-                noSpamLogger.info("{} incompatible schema encountered while deserializing a message", id(), e);
+                noSpamLogger.info("{} incompatible schema encountered while deserializing a message", InboundMessageHandler.this, e);
```

Comment:
self-nit: I thought it might be nice for readability (and future-proofing?) to use `InboundMessageHandler.this` even in the other cases where it's no necessary. Thoughts?